ocs_ci/ocs/ocp.py

```diff
@@ -115,7 +115,19 @@ def exec_oc_cmd(
 
         """
         oc_cmd = "oc "
-        kubeconfig = os.getenv('KUBECONFIG')
+        if os.getenv('KUBECONFIG'):
+            kubeconfig = os.getenv('KUBECONFIG')
+        elif os.path.exists(os.path.join(
+            config.ENV_DATA['cluster_path'],
+            config.RUN.get('kubeconfig_location')
+        )):
```

Comment:
I would define this:  
```
os.path.join(
            config.ENV_DATA['cluster_path'],
            config.RUN.get('kubeconfig_location')
        ) 
```
as a variable to avoid double constructing the same string line bellow.

Reply:
Addressed in 5063216.